static/app/components/charts/barChart.tsx

```diff
@@ -60,13 +60,14 @@ export function transformToBarSeries({
 }
 
 const EMPTY_AXIS = {};
-export function BarChart({
+function BarChartUnwrapped({
```

Comment:
Nit, but I think it's fine to just keep things local and do `export const BarChart = forwardRef(` which also avoid the issue of going to definition taking you to the forwardRef on L92 as opposed to here

Reply:
+ you can avoid the `forwardedRef` type